src/SystemCommands-RefactoringSupport/RBPullUpPreview.class.st

```diff
@@ -0,0 +1,90 @@
+"
+I am a preview of Pull up refactoring, I can select the superClass of pull up methos, and select the methods of class to pull up
+"
+Class {
+	#name : #RBPullUpPreview,
+	#superclass : #RBPushDownPreview,
+	#instVars : [
+		'dropList',
```

Comment:
as I understand dropList is a list of classes. The name classDropList would be much better